ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -632,6 +632,30 @@ Loader {
                         return "";
                     }
```

Comment:
Above here is `messageContent` which used to be for stickers and images. Is it still used for images?
If not, we can remove the conditon.